pkgs/development/haskell-modules/configuration-hackage2nix/broken.yaml

```diff
@@ -1485,6 +1485,7 @@ broken-packages:
   - Feval
   - fez-conf
   - ffeed
+  - fft
```

Comment:
I am not sure why fft was marked broken, it seems to build fine on hydra https://hydra.nixos.org/job/nixpkgs/trunk/haskellPackages.fft.x86_64-linux